<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlat</class>
 <widget class="QDialog" name="AddFlat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление квартиры</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/window_icons/icons/card.png</normaloff>:/new/window_icons/icons/card.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(225, 225, 225);
}

QPushButton{
background: rgb(210,210,210);
border: none;
border-radius: 4px;
font-family: Calibri;
font-size: 10pt;
}
QPushButton:hover{
background: rgb(195, 195, 195);
}

QLineEdit {
border: none;
}

QLabel{
font-size: 11pt;
font-family: Calibri;
}</string>
  </property>
  <widget class="QLineEdit" name="addres_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="block_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="rooms_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="area_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Этаж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во комнат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Площадь</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
